lib/matplotlib/backends/backend_ps.py

```diff
@@ -134,6 +134,77 @@ def _move_path_to_path_or_stream(src, dst):
         shutil.move(src, dst, copy_function=shutil.copyfile)
 
 
+def _font_to_ps_type3(font_path, glyph_ids):
+    font = get_font(font_path, hinting_factor=1)
+
+    preamble = """\
+%!PS-Adobe-3.0 Resource-Font
+%%Creator: Converted from TrueType to Type 3 by Matplotlib.
+20 dict begin
+/_d {{bind def}} bind def
+/_m {{moveto}} _d
+/_l {{lineto}} _d
+/_ce {{closepath eofill}} _d
+/_c {{curveto}} _d
+/_sc {{setcachedevice}} _d
+/_e {{exec}} _d
```

Comment:
Not sure I see any use for this definition?

Reply:
yup, that was just an old leftover